plugins/orchestrator-common/scripts/openapi.sh

```diff
@@ -15,6 +15,7 @@ openapi_generate() {
 
     # Docs generation
     npx --yes @openapitools/openapi-generator-cli@v2.13.4 generate -g markdown -i ${OPENAPI_SPEC_FILE} -o ./src/generated/docs/markdown/
+    npx --yes @openapitools/openapi-generator-cli@v2.13.4 generate -g html2 -i ${OPENAPI_SPEC_FILE} -o ./src/generated/docs
```

Comment:
Ok so you added a new "format" for the docs, I suggest creating a dedicate folder. So `src/generated/docs/html2`.
In this wait we won't mix and match different docs file.

Reply:
done